gosec.sh

```diff
@@ -20,11 +20,6 @@ NC=$(tput sgr0) # No Color
 
 GO_SEC_ARGS=""
 
-VERBOSE=false
-if [[ $* == *verbose* ]]; then
-    VERBOSE=true
-fi
-
```

Comment:
Shellcheck reported that `VERBOSE` is not used, but I don't know if `go sec` use it behind the scenes. Could you confirm?

Reply:
the next `if` was supposed to be not the same as this one, but to check `VERBOSE` variable